packages/components/src/base-control/stories/index.js

```diff
@@ -15,13 +14,14 @@ export default {
 const BaseControlWithTextarea = ( { id, ...props } ) => {
 	return (
 		<BaseControl id={ id } { ...props }>
-			<TextareaControl id={ id } />
+			<textarea style={ { display: 'block' } } id={ id } />
```

Comment:
Is there a particular reason why we'd swap the library's component with the vanilla HTML one? Do you think this should be a pattern that we apply to stories across the package?

Reply:
Haha so when I was working on this PR there was a weird issue I was debugging for half an hour, and I finally realized that `TextareaControl` is **already built with `BaseControl`** 🤦🏻‍♀️ 15095fb9be468299c420de19a3c5cbe5cb9b9bfa